### PR TITLE
Minor spelling fixes, updated booster query

### PR DESCRIPTION
**Pull request by DebitCardz**
You can now specify how many servers you would like to query when using top boosters, example being you can get the top 5 boosted servers or just the top.

### CODE REVIEW

**Review by JeroenoBoy — CHANGES_REQUESTED** [review]
This will break the wrapper, server.js line 9

**Comment by DebitCardz** [comment]
Issue resolved, my mistake.

**Comment by DebitCardz** [comment]
please query that booster alex howadeasra

**Comment by JeroenoBoy** [comment]
After further inspections (aka not on my phone), these changes are inefficient / not required.

File: src/groups/servers.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ module.exports = {
 	 * Amount of servers online
 	 */
 	count: async () => {
-		const { server_count } = await retrieveData();
```
Comment (unlimitedcoder2) [inline src/groups/servers.js:9]: this literally breaks it lol @DebitCardz

**Pull request closed without merging**